deepcell/applications/application.py

```diff
@@ -312,6 +312,47 @@ def _resize_output(self, image, original_shape):
 
         return image
 
+    def _batch_predict(self, tiles, batch_size):
+        """Batch process tiles to generate model predictions.
+
+        The built-in keras.predict function has support for batching, but
+        loads the entire image stack into GPU memory, which is prohibitive
+        for large images. This function uses similar code to the underlying
+        model.predict function without soaking up GPU memory.
+
+        Args:
+            tiles: Tiled data which will be fed to model
+            batch_size: Number of images to predict on per batch
```

Comment:
Don't forget the data type for the args!

```suggestion
            tiles (np.array): Tiled data which will be fed to model
            batch_size (int): Number of images to predict on per batch
```